### ISSUE TITLE
About Learning Rate

### ISSUE THREAD

**Issue by suyeH** [FateScript/CenterNet-better#13]
Hi, this repo is great.
When I was reproducing it, I found that in official CenterNet repo and paper, the base learning rate is 5e-4. But your learning rate is 0.02. Why your learning rate is much larger? And have you ever tried smaller learning rate?

**Comment by FateScript** [comment]
> Hi, this repo is great.
> When I was reproducing it, I found that in official CenterNet repo and paper, the base learning rate is 5e-4. But your learning rate is 0.02. Why your learning rate is much larger? And have you ever tried smaller learning rate?

CenterNet use adam while this repo use sgd, and in COCO training, a standard setting is using lr=0.02 with batchsize=16. This repo use a larger batchsize, which is 128, so this lr is smaller compare with standard setting.

I never tried a smaller learning rate and I believe it might get similar performance.

**Comment by suyeH** [comment]
It seems that lr=5e-4, bs=128 is the standard setting?
![Screenshot from 2020-05-13 14-15-23](https://user-images.githubusercontent.com/23137816/81777817-3a6ffb00-9524-11ea-9d1b-4c5ebe00630f.png)

**Comment by FateScript** [comment]
> It seems that lr=5e-4, bs=128 is the standard setting?
> ![Screenshot from 2020-05-13 14-15-23](https://user-images.githubusercontent.com/23137816/81777817-3a6ffb00-9524-11ea-9d1b-4c5ebe00630f.png)

Yes, and the writer is using adam while I am using sgd. 

Remember that this repo is not a total copy of CenterNet, I modified some part of setting. You may check [this issue](https://github.com/FateScript/CenterNet-better/issues/1) to get more information.

**Comment by suyeH** [comment]
> > It seems that lr=5e-4, bs=128 is the standard setting?
> > ![Screenshot from 2020-05-13 14-15-23](https://user-images.githubusercontent.com/23137816/81777817-3a6ffb00-9524-11ea-9d1b-4c5ebe00630f.png)
> 
> Yes, and the writer is using adam while I am using sgd.
> 
> Remember that this repo is not a total copy of CenterNet, I modified some part of setting. You may check [this issue](https://github.com/FateScript/CenterNet-better/issues/1) to get more information.

Ok, I get it. Thanks for your reply.
